doc/source/users_guide/trouble-shooting/trouble-shooting.rst

```diff
@@ -6,44 +6,121 @@
 Trouble Shooting 
 *********************
 
-In this chapter we give some guidance on what to do when you encounter some of the most common problems. We can't cover all the problems that a user could potentially have, but we will try to help you recognize some of the most common situations. And we'll give you some suggestions on how to approach the problem to come up with a solution.
+In this chapter we give some guidance on what to do when you encounter some of the most common problems.
 
-In general you will run into one of three type of problems:
```

Comment:
Oh, I like the old wording: you WILL run into a problem :-)  (just kidding)

Reply:
Ha! Perhaps: "The odds are good that you will run into a problem" :-)